### ISSUE TITLE
Curl --resolve analog in ffuf

### ISSUE THREAD

**Issue by Rosanna11235813213455** [ffuf/ffuf#657]
Hi!
May I ask if there is a curl "--resolve" analog in ffuf? The point is to check accessibility on another IP, so it should be like DNS response hijacking. Proxy mode or Host header changing are not the options.
Thanks.

**Comment by joohoi** [comment]
Hi, there’s no option for that atm. The best workaround for the time being
is to add the name and IP address to /etc/hosts file.

I might address this in the future while implementing DNS caching and
potentially separating DNS resolving from the normal workflow.

**Comment by Rosanna11235813213455** [comment]
Thank you for the answer!
Yeah, it would be great, thanks.
